### DISCUSSION TITLE
Vue 3 example?

### DISCUSSION THREAD

**Discussion by Rolanddoda** [vercel/turborepo#1502] [Ideas]
It would be nice to see a Vue 3 example sharing ts, eslint and prettier config

**Comment by chris-nccpa** [comment]
I agree, Vue is my primary framework. I should have an example soon as I'm using Turborepo for a new large project with multiple Vue SPAs hosted on one server using Vite, Vue3, Tailwind, and many Nuxt-like features. @Rolanddoda let me know if you still need this.

**Reply by Rolanddoda** [reply]
@chris-nccpa please 🙏 

ps Nuxt-like features like auto-import components & apis, pages, etc sound awesome!

**Reply by mehulkar** [reply]
@chris-nccpa happy to take a PR for this!

**Comment by anthonyshew** [comment]
We have one now!

https://github.com/vercel/turbo/tree/main/examples/with-vue-nuxt
